code/index.tsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const electron = require('electron');
+const app = electron.app;  // Module to control application life.
+const BrowserWindow = electron.BrowserWindow;  // Module to create native browser window.
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+var mainWindow = null;
```

Comment:
Always use let or const.

We should get setup with [tslint](https://github.com/palantir/tslint) to enforce this sort of thing.